rllib/examples/models/autoregressive_action_dist.py

```diff
@@ -0,0 +1,147 @@
+from ray.rllib.models.tf.tf_action_dist import Categorical, ActionDistribution
```

Comment:
All example Models were copied (will be moved in follow-up PR) into the examples/models/ dir and a PyTorch version to each one was added, such that all example scripts will be supporting a `--torch` flag to run the exact same script using PyTorch.